controllers/ebulletin/lib/contact-schema.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+const Joi = require('@hapi/joi16');
+
+function getTranslations(i18n) {
+    return function (path, ...params) {
+        return i18n && i18n.__(`toplevel.ebulletin.${path}`, ...params);
+    };
+}
+
+function buildValidLocations (i18n) {
+    const labelForLocale = getTranslations(i18n);
+    return [
+        {
+            value: 'england',
+            label: labelForLocale('locations.england'),
+        },
+        {
+            value: 'northern-ireland',
+            label: labelForLocale('locations.northernIreland'),
+        },
+        {
+            value: 'scotland',
+            label: labelForLocale('locations.scotland'),
+        },
+        {
+            value: 'wales',
+            label: labelForLocale('locations.wales'),
+        },
+    ];
+}
+
+module.exports = {
+    buildValidLocations,
+    newContact(i18n) {
+        const messageForLocale = getTranslations(i18n);
+        const locations = buildValidLocations(i18n);
+
+        const email = Joi.string().email().required();
+        const firstName = Joi.string().required();
+        const lastName = Joi.string().required();
+        const location = Joi.string().valid(...locations.map(_ => _.value)).required();
+        const organisation = Joi.string().allow('').optional();
+
+        return {
+            schema: Joi.object({
```

Comment:
Mind adding a couple of small unit tests for the schema? Like we do for account schemas. This isn't complex validation logic so just a test or two.